src/dc_context.c

```diff
@@ -392,6 +399,9 @@ static char* get_sys_config_str(const char* key, const char* def)
  * - `server_flags` = IMAP-/SMTP-flags as a combination of @ref DC_LP flags, guessed if left out
  * - `displayname`  = Own name to use when sending messages.  MUAs are allowed to spread this way eg. using CC, defaults to empty
  * - `selfstatus`   = Own status to display eg. in email footers, defaults to a standard text
+ * - `selfavatar`   = File containing avatar.
+ *                    Will be copied to blob directory, if needed.
```

Comment:
what does the "if needed" mean?  I'd expect it to always get copied (and if it's there already, not copying is fine, but implementation detail optimization).